visualizations/frontend/other/TTable.vue

```diff
@@ -240,6 +240,13 @@
         @setResetFunction="setPagerResetFunction"
       />
     </div>
+    <slot
+      name="footer"
+      :total-count="totalRows"
+      :visible-count="rowsPerPage"
+      :start-ondex="startIndex"
```

Comment:
should this be start-index instead of start-ondex?

Reply:
@msorber thanks for catch, corrected it.